### ISSUE TITLE
bind() function cannot handle empty attribute

### ISSUE THREAD

**Issue by del680202** [cannatag/ldap3#467]
Hello, I am going to use ``ldap3`` to validate user/password.

My environment:
* CentOS 7.4
* ldap3=2.4

I test this library as below:
```
import ldap3
conn = ldap3.Connection('ldap://myldap.com:389', user='uid=chintzulin01,ou=people,o=example.com', password='mypassword')
conn.bind()
```

Then it throw exception
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python2.7/site-packages/ldap3/core/connection.py", line 594, in bind
    self.refresh_server_info()
  File "/usr/lib/python2.7/site-packages/ldap3/core/connection.py", line 1315, in refresh_server_info
    self.server.get_info_from_server(self)
  File "/usr/lib/python2.7/site-packages/ldap3/core/server.py", line 446, in get_info_from_server
    self._get_schema_info(connection)
  File "/usr/lib/python2.7/site-packages/ldap3/core/server.py", line 431, in _get_schema_info
    self._schema_info.other[attribute] = format_attribute_values(self._schema_info, attribute, self._schema_info.raw[attribute], self.custom_formatter)
  File "/usr/lib/python2.7/site-packages/ldap3/protocol/formatters/standard.py", line 200, in format_attribute_values
    formatted_values = [formatter(raw_value) for raw_value in values]  # executes formatter
TypeError: 'NoneType' object is not iterable
```

I checked ``self._schema_info``, there are 3 attributes
```
(u'cn', ['Subschema'])
(u'objectClass', ['top', 'subentry', 'subschema', 'extensibleObject'])
(u'controls', None)
```

``controls`` attribute is empty then method ``format_attribute_values`` cannot handle it.

Currently, I just added hard-code as bellow to escape this problem.
```python standard.py
def format_attribute_values(schema, name, values, custom_formatter):

    if values is None:
        return []
...
```

Any advice?

Thanks

**Comment by cannatag** [comment]
This should not happen, LDAP protocol specifies that a value must always be returned by the server. Anyway I've added your suggested checking. Can you try the code in dev to see if it works as expected?

Thanks,
Giovanni

**Comment by del680202** [comment]
Checked! It works well, thanks

**Comment by cannatag** [comment]
Good, the fix will be released in the next version of ldap3

**Comment by cannatag** [comment]
Released in version 2.4.1
